### DISCUSSION TITLE
Disable day/night mode by default on Android 10+

### DISCUSSION THREAD

**Discussion by papjul** [breezy-weather/breezy-weather#1197] [Ideas]
### Describe your idea

By default, day/night mode is enabled, meaning the theme on main page will be light when the sun is up in the chosen location, and dark when the sun is down.

However, since Android 10, there is a native alternative. The user can either from Android settings: force light theme all the time, force dark theme all the time, or make the theme follow sunrise/set.

For this reason, I believe only Android 10+ users who enabled the latter would be interested in having this mode enabled by default. For users always using dark theme for example, opening the app and having straight away a light theme is confusing, and some users might not know how to change this.

Disabling this setting would make the app follow Android settings preferences.

For Android < 10 with no native dark theme, I believe it should remain enabled by default.

This is open to discussion. Let me know what you think, either thumb up/down or discuss below :)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by min7-i** [comment]
I like that idea. That brings the app's behavior more in line with other apps.

However, I'm wondering if it makes sense to keep this setting enabled for Android 10 and lower. The argument "[that] some users might not know how to change this" also applies to those versions.

**Reply by papjul** [reply]
> However, I'm wondering if it makes sense to keep this setting enabled for Android 10 and lower. The argument "[that] some users might not know how to change this" also applies to those versions.

On Android < 10, disabling this option by default would result in making it light theme all the time. Making it bright during the day and dark during the night is a better default behavior than "always light" IMO
